stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/event/EventBatchUpdate.kt

```diff
@@ -43,8 +43,12 @@ internal class EventBatchUpdate private constructor(
             channel.updateLastMessage(message)
 
             val currentUserId = domainImpl.user.value?.id
```

Comment:
What do you think about replacing it with?
```
            val currentUserId = domainImpl.user.value?.id
            if (isNewMessage && currentUserId != null) {
                val lastReadDate = channel.read.firstOrNull { it.user.id == currentUserId }?.lastMessageSeenDate
              
                if (message.shouldIncrementUnreadCount(currentUserId, lastReadDate)) {
                    channel.incrementUnreadCount(currentUserId, message)
                }
            }
```
We don't need to go through `channel.read` if the message is not new or currentUserId != null

Reply:
Ok, it is a bit faster indeed.